experimental/traffic-portal/src/app/core/cache-groups/cache-group-table/cache-group-table.component.ts

```diff
@@ -165,18 +165,13 @@ export class CacheGroupTableComponent implements OnInit {
 		},
 		{
 			href: "/core/asns",
-			name: "Manage ASNs",
+			name: "View ASNs",
```

Comment:
This isn't from your changes but if you could also fix the `Edit`/`Open in New Tab` buttons leading to an invalid route: `core/cache-groups/core/cache-groups/2`